src/popup/router/components/Modals/TransferReceive.vue

```diff
@@ -0,0 +1,294 @@
+import Truncate from '../Truncate.vue';
+import Button from '../Button.vue';
+
+export default {
+  name: 'TransferReceive',
+  components: {
+    RequestAmount,
+    Modal,
+    Avatar,
+    Truncate,
+    QrcodeVue,
+    Button,
+    Scrollable,
+  },
+  mixins: [
+    CopyMixin,
+  ],
+  props: {
+    accountIdx: { type: Number, default: -1 },
+  },
+  data() {
+    return {
+      shareIcon,
+      amount: null,
+      IS_MOBILE_DEVICE: window.IS_MOBILE_DEVICE,
+      selectedAsset: null,
+    };
+  },
+  computed: {
+    ...mapState('accounts', ['activeIdx']),
+    ...mapGetters(['accounts', 'activeNetwork']),
+    idx() {
+      return this.accountIdx === -1 ? this.activeIdx : this.accountIdx;
+    },
+  },
+  methods: {
+    getTokenInfo(link = false) {
+      if (this.amount <= 0) return '';
+      const token = this.selectedAsset && (this.selectedAsset.symbol === 'AE'
+        ? 'AE'
+        : this.selectedAsset.contractId);
+      const firstChar = link ? '&' : '?';
+      const separator = link ? '&' : '&amp;';
+      return `${firstChar}token=${token}${separator}amount=${this.amount}`;
+    },
+    getLink(value) {
+      return `${APP_LINK_WEB}/transfer?account=${value}${this.getTokenInfo(true)}`;
+    },
+    async share() {
+      const { address } = this.accounts[this.idx];
+      const walletLink = this.getLink(address);
+      const text = (this.amount > 0)
+        ? this.$t('modals.receive.shareTextNoAmount', { address, walletLink })
+        : this.$t('modals.receive.shareTextWithAmount', { address, walletLink, amount: this.amount });
+      await this.$store.dispatch('share', { text });
+    },
+    getQRdata(address) {
+      return this.amount > 0
+        ? `${this.getLink(address)}`
+        : address;
+    },
+    truncateAddress(address) {
+      return address.match(/.{1,3}/g).reduce((acc, current) => `${acc} ${current}`);
+    },
+    formatAddress(address) {
+      return this.amount > 0
+        ? `${address}?${this.getTokenInfo(true).substring(1)}`
+        : this.truncateAddress(address);
+    },
+    getTextToCopy() {
+      return this.amount > 0
+        ? this.getLink(this.accounts[this.idx].address)
+        : this.accounts[this.idx].address;
+    },
+    explorerUrl() {
+      const { address } = this.accounts[this.idx];
+      return `${this.activeNetwork.explorerUrl}/account/transactions/${address}`;
+    },
+    handleAssetSelection(newToken) {
+      this.selectedAsset = newToken;
+    },
+    closeModal() {
+      this.$store.commit('modals/closeByKey', MODAL_TRANSFER_RECEIVE);
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+@use '../../../../styles/variables';
+@use '../../../../styles/typography';
+@use '../../../../styles/share-info';
+@use '../../../../styles/mixins';
+
+.transfer-receive-modal {
+  font-weight: 500;
+  color: variables.$color-white;
+
+  .transfer-receive {
+    display: flex;
+    align-content: center;
+    justify-content: center;
+    flex-direction: column;
```

Comment:
You can use mixins.flex here

Reply:
After some tests it turns out that the flex definitions serves no purpose here so I totally removed this block of code.